environment_acceptance.sh

```diff
@@ -1,3 +1,3 @@
 export SETTINGS='config.TestConfig'
 export PPI_END_POINT='http://landregistry.local:8002/get_ppi'
-export POSTGRES_DATABASE_URI='postgresql+pg8000://postgres:seekritpwd@172.16.42.43:5432/pubdata'
+export ELASTIC_SEARCH_ENDPOINT='http://ec2-54-77-238-128.eu-west-1.compute.amazonaws.com:4200'
```

Comment:
think there are different ports for environment and environment_acceptance, see @AndrewGorton's note:
```
http://ec2-54-77-238-128.eu-west-1.compute.amazonaws.com:4300/landregistry/property/_search?q=3_MADISON_AVENUE_EXETER_EX1_3AH

Andrew Gorton [10:41 AM]
Note port is 4300

Andrew Gorton [10:41 AM]
4300 => Accept

Andrew Gorton [10:41 AM]
http://ec2-54-77-238-128.eu-west-1.compute.amazonaws.com:4200/landregistry/property/_search?q=3_MADISON_AVENUE_EXETER_EX1_3AH

Andrew Gorton [10:41 AM]
4200 => Integration
```